src/__tests__/looksrare/orderValidator.test.ts

```diff
@@ -8,12 +8,16 @@ import { SupportedChainId, CollectionType, StrategyType, CreateMakerInput, Order
 describe("Order validation", () => {
   let mocks: SetupMocks;
   let signers: Signers;
+  let lrUser1: LooksRare;
   let baseMakerAskInput: CreateMakerInput;
   let baseMakerBidInput: CreateMakerInput;
+
   beforeEach(async () => {
     mocks = await setUpContracts();
     signers = await getSigners();
 
+    lrUser1 = new LooksRare(SupportedChainId.HARDHAT, ethers.provider, signers.user1, mocks.addresses);
```

Comment:
It wouldn't save much code but given how often you repeat this I wonder if it could be put into a `getLooksRareUser()` helper.

Reply:
I'd need to pull the signers and contracts, it's a bit over engineered